packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.ts

```diff
@@ -70,19 +70,18 @@ export abstract class XYChartComponent extends TimeseriesChartComponent
         return this.configuration.legendPlacement === LegendPlacement.Right;
     }
 
-    public onLegendClick(legendSeries: IChartAssistSeries<IAccessors>, event: MouseEvent) {
+    public onPrimaryDescClick(event: any, legendSeries: IChartAssistSeries<IAccessors>) {
```

Comment:
It's better to put a type here if we can. This will likely make your unit test need an `as MouseEvent` after the argument value.

```suggestion
    public onPrimaryDescClick(event: MouseEvent, legendSeries: IChartAssistSeries<IAccessors>) {
```